src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -107,48 +130,30 @@ export default {
 		onLoad() {
 			this.apollo.watchQuery({ query: publicLendMenuQuery }).subscribe({
 				next: ({ data }) => {
-					this.userId = _get(data, 'my.userAccount.id');
```

Comment:
This still needs to be queried for as the user may become logged by using the pop-up login form the sign-in button in the header

Reply:
See my comment above

Reply:
Turns out you're already still querying for this because you've moved it into the apollo.result function, which has a watchQuery automatically setup for it and thus is responding to any changes in the logged-in state, my mistake.